content/guides/kubernetes/velero-gs.md

```diff
@@ -0,0 +1,389 @@
+
+    ```bash
+    kubectl -n ghost get all
+
+    NAME                         READY   STATUS    RESTARTS   AGE
+    pod/ghost-86b56c9d44-gk2f8   1/1     Running   0          44m
+    pod/ghost-mariadb-0          1/1     Running   0          50m
+
+    NAME                    TYPE           CLUSTER-IP     EXTERNAL-IP     PORT(S)        AGE
+    service/ghost           LoadBalancer   10.108.8.228   35.233.219.60   80:31603/TCP   50m
+    service/ghost-mariadb   ClusterIP      10.108.8.42    <none>          3306/TCP       50m
+
+    NAME                    READY   UP-TO-DATE   AVAILABLE   AGE
+    deployment.apps/ghost   1/1     1            1           44m
+
+    NAME                               DESIRED   CURRENT   READY   AGE
+    replicaset.apps/ghost-86b56c9d44   1         1         1       44m
+
+    NAME                             READY   AGE
+    statefulset.apps/ghost-mariadb   1/1     50m
+    ```
+
+7. Now open your `Admin URL` in a web browser. If it doesn't load, double-check that the pods shown in the previous section show up as `1/1` in the `READY` column. Sign in the admin interface with the email and password that we entered earlier in the values file. You should now see the following screen, which will let us create a blog:
+   ![Ghost Admin Page](/images/guides/kubernetes/velero-gs/1-welcome-to-ghost.png)
+
+   Click on the `+` next to Posts to create your first post. Enter a title and some content, then click Publish in the top right.
+   ![Ghost Publish Page](/images/guides/kubernetes/velero-gs/2-ghost-create-post.png)
+
+   The home page should now look like this:
+   ![Ghost Page With Post](/images/guides/kubernetes/velero-gs/3-ghost-home-page.png)
+
+## Backup
+Now, let’s look into backups in case something happens to the blog. Backups are for Kubernetes resources and persistent volumes. You can back up your entire cluster, or optionally choose a namespace or label selector to back up. 
+
+They can be run one off or scheduled. It’s a good idea to have scheduled backups so you are certain you have a recent backup to easily fall back to. You can also create [backup hooks](https://velero.io/docs/v1.5/backup-hooks/) if you want to execute actions before or after a backup is made.
+
+By default, the backup retention is 30 days, but you can change it with the `--ttl` flag.
+
+
+1. [Optional] To see what all you can do with backups run:
+    ```bash
+    velero backup -h
+    ```
+2. Let’s start with the most basic option: creating a one off backup. It’s a good idea to give it a meaningful name so you remember what it was for unlike what is done here.
+    ```bash
+    BACKUP=gastly
+    velero backup create $BACKUP --include-namespaces ghost
+    ```
+
+    If we wanted to do a backup with a specific namespace or to exclude a specific namespace we can do:
```

Comment:
Try make it clearer not to run these commands ...  I was quickly going through the doc and almost copy and pasted them.